tests/cancun/eip6780_selfdestruct/test_double_selfdestruct_revert.py

```diff
@@ -0,0 +1,182 @@
+"""
+Suicide scenario requested test
+https://github.com/ethereum/tests/issues/1325
+"""
+
+from collections import namedtuple
+from typing import Any, Mapping, Optional
+
+import pytest
+from ethereum_test_forks import (
+    Berlin,
+    Cancun,
+    Fork,
+    Frontier,
+    Homestead,
+    London,
+    Merge,
+    Shanghai,
+    is_fork
+)
```

Comment:
```suggestion
from ethereum_test_forks import Cancun, Fork
```
I think most of the forks imported are not actually used.

`is_fork` was also just removed in https://github.com/ethereum/execution-spec-tests/pull/367 in favor of the operators >, >=, < and <=.